base/inc/lessc.inc.php

```diff
@@ -1,3675 +1,276 @@
 <?php
-
 /**
- * lessphp v0.4.0
- * http://leafo.net/lessphp
- *
- * LESS css compiler, adapted from http://lesscss.org
- *
- * Copyright 2012, Leaf Corcoran <leafot@gmail.com>
- * Licensed under MIT or GPLv3, see LICENSE
+ * This file provides the part of lessphp API (https://github.com/leafo/lessphp)
+ * to be a drop-in replacement for following products:
+ *  - Drupal 7, by the less module v3.0+ (https://drupal.org/project/less)
+ *  - Symfony 2
  */
 
+// Register autoloader for non-composer installations
+if ( !class_exists( 'Less_Parser' ) ) {
+	require_once __DIR__ . '/lib/Less/Autoloader.php';
```

Comment:
I think `__DIR__ ` isn't PHP 5.2 compatible, which we're still supporting for the time being.